public/app/features/alerting/unified/utils/cloud-alertmanager-notifier-types.ts

```diff
@@ -384,6 +384,10 @@ export const cloudNotifierTypes: Array<NotifierDTO<CloudNotifierType>> = [
         required: true,
         setValueAs: (value) => (typeof value === 'string' ? parseInt(value, 10) : 0),
       }),
+      option("message_thread_id", "Message thread ID", "ID of the message thread where to send the messages", {
+        required: false,
+        setValueAs: (value) => (typeof value === 'string' ? parseInt(value, 10) : 0),
+      }),
```

Comment:
this can be removed until PR in Alertmanager is merged.

Reply:
I removed this part, but didn't fully understand what you meant. Should I open another PR later for this part?